src/cli.cpp

```diff
@@ -58,6 +70,30 @@ void HelpAndQuit(cxxopts::Options options)
     exit(0);
 }
 
+string filename = "plot.dat";
+string tempdir = ".";
+string tempdir2 = ".";
+
+void cleanup(const string& tmpdir)
+{
+    string glob = fs::path(tmpdir) / (filename + ".*.tmp");
+    cout << "Cleaning up " << glob << " .." << endl;        
+    for (const auto& f : fs::directory_iterator(tmpdir)) {
+        const auto fname = f.path().filename().string();
+        if (f.is_regular_file() && startsWith(fname, filename) && endsWith(fname, ".tmp")) {
```

Comment:
this assumption of filenames may be correct in 99% of the cases, but the time it *isn't* correct, it can have quite bad consequences. I think being accurate in which files to remove is important.
Consider running two plots in parallel for instance, sharing the same temp directory.

Reply:
If two plots run in parallel in same tmpdir, `filename` will still be different, right?

Reply:
you might be right. It may have been a poor example. Maybe this rule is correct in more than 99% of cases, but you can still construct cases where it would delete files unrelated to the plot.

Reply:
I see what you mean: if using CLI, sure, if you name your first file `plot`, your second one `plot.x` and you kill the `plot` process, it will accidentally remove `plot.x.*` too. In that case, bad luck.

When using the DiskPlotter from `chia plots create`, which is the normal case, such scenario is not currently possible.